go-app-ussd_covid19_triage.js

```diff
@@ -258,6 +431,36 @@ go.app = (function () {
       });
     });
 
+    self.add("state_age_years", function (name) {
+      if (self.im.user.answers.state_age_years && self.im.user.answers.state_age) {
+        return self.states.create("state_province");
+      }
+      var question = $("Please TYPE your age in years (eg. 35)");
+      return new FreeText(name, {
+        question: question,
+        check: function(content) {
+          var age = _.parseInt(content);
+          if(age < 1) {
+            return question;
+          } else if (age < 18) {
+            self.im.user.answers.state_age = "<18";
+            return;
+          } else if (age < 40) {
+            self.im.user.answers.state_age = "18-40";
+            return;
+          } else if (age <= 65) {
+            self.im.user.answers.state_age = "40-65";
+            return;
+          } else if (age < 150) {
```

Comment:
Why not just > 65?

Reply:
I wanted to avoid bad data being accidentally added. If they accidentally type an extra number or something, eg. they type `566` instead of `56`

Reply:
+1